tests/dragonfly/snapshot_test.py

```diff
@@ -160,6 +160,26 @@ async def test_snapshot(self, df_seeder_factory, df_server):
         assert super().get_main_file("test-periodic-summary.dfs")
 
 
+@dfly_args({**BASIC_ARGS, "dbfilename": "test-periodic", "snapshot_cron": "*/2 * * * * *"})
+class TestCronPeriodicSnapshot(SnapshotTestBase):
+    """Test periodic snapshotting"""
+
+    @pytest.fixture(autouse=True)
+    def setup(self, tmp_dir: Path):
+        super().setup(tmp_dir)
+
+    @pytest.mark.asyncio
+    async def test_snapshot(self, df_seeder_factory, df_server):
+        seeder = df_seeder_factory.create(
+            port=df_server.port, keys=10, multi_transaction_probability=0
+        )
+        await seeder.run(target_deviation=0.5)
+
+        time.sleep(60)
```

Comment:
Is there any way to avoid such long sleeps? That will one day (hopefully soon) delay submissions

Reply:
At first I set it to 30 seconds but the test occasionally failed,  maybe due to initiation processes taking time as well (I ran this test case alone some of the time).

Reply:
I see that the above test also sleeps for 60s, so while not great, let's try to address that in a future PR. Hopefully the change to run tests in parallel will reduce the need for that.